src/translator/vocabs.h

```diff
@@ -0,0 +1,81 @@
+#pragma once
+
+namespace marian {
+namespace bergamot {
+
+/// Wrapper of Marian Vocab objects needed for translator.
+/// Holds multiple source vocabularies and one target vocabulary
+class Vocabs {
+public:
+  /// Construct vocabs object from either byte-arrays or files
+  Vocabs(Ptr<Options> options, std::vector<std::shared_ptr<AlignedMemory>>&& vocabMemories): options_(options){
+    if (!vocabMemories.empty()){
+      // load vocabs from buffer
+      load(std::move(vocabMemories));
+    }
+    else{
+      // load vocabs from file
+      auto vocabPaths = options->get<std::vector<std::string>>("vocabs");
+      load(vocabPaths);
+    }
+  }
+
+  /// Get all source vocabularies (as a vector)
+  std::vector<Ptr<Vocab const>>& source(){
+    return srcVocabs_;
+  }
```

Comment:
nit perhaps - `sources()` instead of `source()`. We don't want these (`sources()`, `target()`) to be edited outside this class. Can we get away with making these `const` references?